prob-6/prob6.py

```diff
@@ -0,0 +1,6 @@
+s1 = s2 = 0
+
+for i in range(1, 101):
+    s1 += i ** 2
+    s2 += i
```

Comment:
This for loop is better written by first collecting the elements, then using `sum` function